src/descriptor/table_schema.rs

```diff
@@ -1655,6 +1657,28 @@ impl From<Vec<String>> for TableSchemaObjectPrimaryKey {
         Self::Variant0(value)
     }
 }
+impl Display for TableSchemaObjectPrimaryKey {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            TableSchemaObjectPrimaryKey::Variant0(fields) => {
+                f.write_str("(")?;
+                let mut iter = fields.iter().peekable();
```

Comment:
Would the following not work? 

```suggestion
                f.write_str(fields.join(", "));
```

Reply:
Oof, I always forget about this! The indirectness makes this less discoverable:
- `Vec<T>` implements `Deref<Target = [T]>`
- Any slice of string-like values (`impl Borrow<str>`) can be `.join`ed with a `&str` separator to produce a `String` via this impl: https://doc.rust-lang.org/src/alloc/str.rs.html#71